docs/faq.md

```diff
@@ -1,154 +1,9 @@
 # FAQ
 
```

Comment:
The FAQ is still the right spot for common questions.

Reply:
We had discussed in standup removing most questions from faq and moving them to more appropriate places in docs